esphome/components/tem3200/tem3200.cpp

```diff
@@ -0,0 +1,154 @@
+#include "tem3200.h"
+#include "esphome/core/log.h"
+#include "esphome/core/helpers.h"
+#include "esphome/core/hal.h"
+
+namespace esphome {
+namespace tem3200 {
+
+static const char *const TAG = "tem3200";
+
+enum ErrorCode {
+  NONE = 0,
+  RESERVED = 1,
+  STALE = 2,
+  FAULT = 3,
+};
+
+void TEM3200Component::setup() {
+  ESP_LOGCONFIG(TAG, "Setting up TEM3200...");
```

Comment:
is TEM3200 redundant?  (will it be displayed here anyhow..  or should other outputs probably say TEM3200?)

Reply:
Yes, it is redundant (as the driver filename will also be in the log). However, this is what the rest of the drivers do. So I copied the pattern. That being said, I think including the device id (and maybe i2c bus) would be a good idea.